modules/network/src/main/java/org/apache/ignite/internal/network/serialization/UnsafeFieldAccessor.java

```diff
@@ -0,0 +1,188 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.ignite.internal.network.serialization;
+
+import java.lang.reflect.Field;
+import org.apache.ignite.internal.util.GridUnsafe;
+
+/**
+ * {@link FieldAccessor} implementation.
+ */
+class UnsafeFieldAccessor implements FieldAccessor {
+    private final Field field;
+    private final Class<?> fieldType;
+    private final long fieldOffset;
+
+    UnsafeFieldAccessor(FieldDescriptor descriptor) {
+        field = findField(descriptor);
```

Comment:
We create an accessor during the instantiation of the FieldDescriptor where we have `Field`. So maybe we should just pass it as an argument instead of searching for it?

Reply:
It doesn't look too bad, but probably it looks cleaner if we don't search for a Field twice. Fixed.